ligo/src/permit.mligo

```diff
@@ -51,3 +51,13 @@ let get_vote_permit_counter (param, store : vote_permit_counter_param * storage)
     :: ([] : operation list)
   , store
   )
+
+let get_total_supply (param, store : get_total_supply_param * storage) : return =
+  let result = match Big_map.find_opt param.token_id store.total_supply with
+      None ->
+        (failwith("FA2_TOKEN_UNDEFINED") : nat)
+    | Some v -> v in
+  ( Tezos.transaction result 0mutez param.callback
+    :: ([] : operation list)
+  , store
+  )
```

Comment:
```suggestion
  match Big_map.find_opt param.token_id store.total_supply with
    | None -> (failwith("FA2_TOKEN_UNDEFINED") : return)
    | Some v ->
      ( Tezos.transaction result 0mutez param.callback
        :: ([] : operation list)
      , store
      )
```
I'm surprised that the `:: ([] :operation list)` works BTW, shouldn't `Tezos.transaction` be wrapped in `[]`?

Reply:
Actually `::` here is a con, not type specifying. Could be quite confusing with the haskell syntax.

Reply:
Oh yeah :sweat_smile: syntax confusion